evap/evaluation/models_logging.py

```diff
@@ -41,10 +41,11 @@ class LogJSONEncoder(JSONEncoder):
     As JSON can't store datetime objects, we localize them to strings.
     """
 
-    def default(self, obj):
-        if isinstance(obj, (date, time, datetime)):
-            return localize(obj)
-        return super().default(obj)
+    def default(self, o):
+        """ o is an object instance """
```

Comment:
why?

Reply:
`signature-differs` triggered. In the parent class, the argument is named `o`, not `obj`. I forgot to undo this when I disabled `signature-differs`. (Kind of makes sense though, substitution principle and keyword arguments - a call with `instance.default(o=False)` wouldn't work with this child class)